src/components/RightPane/CoursePane/CourseRenderPane.js

```diff
@@ -89,17 +91,44 @@ const flattenSOCObject = (SOCObject) => {
     }, []);
 };
 
-const RecruitmentBanner = ({ className }) => (
-    <div className={className}>
-        Interested in web development?
-        <br />
-        <a href="https://forms.gle/v32Cx65vwhnmxGPv8" target="__blank" rel="noopener noreferrer">
-            Join ICSSC and work on AntAlmanac and other projects!
-        </a>
-        <br />
-        We have positions for experienced devs and those with zero experience!
-    </div>
-);
+const RecruitmentBanner = ({ className }) => {
+    // Idk how else to force a function component to update
+    const [, updateState] = React.useState();
+    const forceUpdate = React.useCallback(() => updateState({}), []);
+
+    let displayRecruitmentBanner =
+        !Boolean(window.localStorage.getItem('dismissedRecruitmentBanner')) &&
+        ['COMPSCI', 'IN4MATX', 'I&C SCI', 'STATS'].includes(RightPaneStore.getFormData().deptValue);
+
+    const bannerStyle = { flexDirection: 'left', justifyContent: 'flex-end' };
+
+    return displayRecruitmentBanner ? (
+        <>
+            <Paper elevation={1} square>
+                <Grid container style={bannerStyle} className={className}>
+                    <div style={{ paddingLeft: '30px' }}>
+                        Interested in web development?
+                        <br />
+                        <a href="https://forms.gle/v32Cx65vwhnmxGPv8" target="__blank" rel="noopener noreferrer">
+                            Join ICSSC and work on AntAlmanac and other projects!
+                        </a>
+                        <br />
+                        We have positions for experienced devs and those with zero experience!
+                    </div>
+
+                    <Button
+                        onClick={() => {
+                            window.localStorage.setItem('dismissedRecruitmentBanner', true);
+                            forceUpdate();
```

Comment:
Then down here we can just set the display state to false
```suggestion
                            setDisplayRecruitmentBanner(false);
```